### PR TITLE
Add update todolist using create todolist fork as base

### PR DESCRIPTION
**Pull request by MarceloBD**
Add update to front and map mutation to backend schema. 
I've used CreateList fork to implement this functionality, so it's important to merge that feature first.

### CODE REVIEW

**Comment by MarceloBD** [comment]
#24

**Comment by MarceloBD** [comment]
This PR refers to issue  #24

File: backend/src/infra/api/graphql/resolvers/mutation/updateList.js [hunk 1]
```diff
@@ -0,0 +1,21 @@
...
+const dependency = {
+  updateList: require('../../../../../domain/usecases/updateList').updateList,
+}
+
+const resolvers = {
+  Mutation: {
+    updateList: async (parent, args) => {
+      const di = Object.assign({}, dependency, args.injection)
+      const uc = di.updateList(args.injection)
+      const hasAccess = uc.authorize({ canCreateList: true }) // TODO: authorize user
```
Comment (dalssoft) [inline backend/src/infra/api/graphql/resolvers/mutation/updateList.js:12]: `canUpdateList`

File: frontend/.eslintrc.json [hunk 2]
```diff
@@ -1,23 +1,36 @@
...
-  "parserOptions": {
-    "ecmaVersion": 2018
-  },
-  "rules": {
-    "semi": ["error", "never"],
-    "func-names": 0,
-    "no-param-reassign": 0,
-    "no-multi-str": 0,
-    "consistent-return": 0
+    "parser": "babel-eslint",
```
Comment (dalssoft) [inline frontend/.eslintrc.json:2]: this kind of file is out of scope for this PR

File: frontend/.prettierrc [hunk 3]
```diff
@@ -1,4 +1,5 @@
 {
-  "singleQuote": true,
-  "trailingComma": "es5"
-}
+    "trailingComma": "es5",
```
Comment (dalssoft) [inline frontend/.prettierrc:2]: this kind of file is out of scope for this PR

**Comment by dalssoft** [comment]
@jhomarolo could you check the frontend files? tks

**Comment by jhomarolo** [comment]
Hello @MarceloBD Marcelo, this PR is breaking due to branch conflicts.

Can you update it please for review?

File: backend/src/infra/api/graphql/resolvers/mutation/updateList.js (hunk 1 above)
Comment (MarceloBD) [reply backend/src/infra/api/graphql/resolvers/mutation/updateList.js]: done

File: frontend/.eslintrc.json (hunk 2 above)
Comment (MarceloBD) [reply frontend/.eslintrc.json]: I removed

File: frontend/.prettierrc (hunk 3 above)
Comment (MarceloBD) [reply frontend/.prettierrc:2]: It is already in file, isnt?
